proto/atomone/gov/v1/genesis.proto

```diff
@@ -30,4 +30,11 @@ message GenesisState {
   //
   // Since: cosmos-sdk 0.47
   Params params = 8;
+  // The constitution allows builders to lay a foundation and define purpose.
+  // This is an immutable string set in genesis.
+  // There are no amendments, to go outside of scope, just fork.
+  // constitution is an immutable string in genesis for a chain builder to lay out their vision, ideas and ideals.
```

Comment:
This comment is irrelevant for AtomOne, the string is mutable as we will have amendments.